rotkehlchen/tests/api/test_uniswap.py

```diff
@@ -560,7 +219,7 @@ def test_get_events_history_get_all_events_empty(
 def test_get_v3_balances_premium(rotkehlchen_api_server):
     """Check querying the uniswap balances v3 endpoint works."""
     response = requests.get(
-        api_url_for(rotkehlchen_api_server, 'uniswapv3balancesresource'),
+        api_url_for(rotkehlchen_api_server, 'modulebalancesresource', module='uniswap_v3'),
```

Comment:
Okay last time I mention it. But since you touch them, why not VCR them **and** remove the flakiness shit?

I won't mention it any further in the review but it should mostly apply for all other tests